resources/istio/charts/kyma/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
```

Comment:
Maybe this should be a part of isito init an label namespace before anything is applied to istio system etc?
And please change name, it is confusing